resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesTaskSetManagerSuite.scala

```diff
@@ -21,12 +21,13 @@ import scala.collection.mutable.ArrayBuffer
 import io.fabric8.kubernetes.api.model.{Pod, PodSpec, PodStatus}
 import org.mockito.Mockito._
 
-import org.apache.spark.{SparkContext, SparkFunSuite}
+import org.apache.spark.{SparkConf, SparkContext, SparkFunSuite}
 import org.apache.spark.scheduler.{FakeTask, FakeTaskScheduler, HostTaskLocation, TaskLocation}
 
 class KubernetesTaskSetManagerSuite extends SparkFunSuite {
 
-  val sc = new SparkContext("local", "test")
+  val sc = new SparkContext(master = "local", appName = "test",
+    new SparkConf().set("spark.driver.allowMultipleContexts", "true"))
```

Comment:
In this project, we have `KubernetesTaskSetManagerSuite` and `KubernetesTaskSchedulerImplSuite`. The two suites run inside the single JVM. We are still running one suite at a time sequentially. But we are creating two `SparkContext` instances. Without this option, the second suite fails because only one `SparkContext` instance is allowed. I was hoping to prevent that. 

But I think there is a better option, which is just to call `SparkContext.clearActiveContext()`. Switched to that.